Readme-podman.md

```diff
@@ -4,12 +4,10 @@ If you have podman configured and running in your environment, then you may
 use it to build your containers with the following:
 
 ```bash
-export DOCKER=podman
+export CONTAINER_TOOL=podman
```

Comment:
Have you tried using podman for this at all?

Reply:
Not recently.